### DISCUSSION TITLE
No Device Role Option Menu

### DISCUSSION THREAD

**Discussion by perdingus** [nautobot/nautobot#4749] [Q&A] [answered]
Hi, new install of nautobot. I don't have a menu item for 'Roles' under the Devices->Device Types  sub menu. I only have 'Device Types' and 'Manufactures' listed.  Is there something I haven't enabled? Thanks in advance.
![roles](https://github.com/nautobot/nautobot/assets/111276207/b834c763-f9ba-499e-a9e1-3e4508fd5121)

**Comment by whitej6** [comment, marked as answer]
In nautobot v2.X roles are now using a common model for Device IPAM etc. The new role is located under the `Organization` menu

**Comment by perdingus** [comment]
Thank you. How silly do I feel!?

**Reply by coflin** [reply]
I feel they should update their documentation (https://docs.nautobot.com/projects/core/en/stable/user-guide/feature-guides/getting-started/creating-devices/) with the same
